django/conf/__init__.py

```diff
@@ -104,6 +104,8 @@ def configure(self, default_settings=global_settings, **options):
             raise RuntimeError('Settings already configured.')
         holder = UserSettingsHolder(default_settings)
         for name, value in options.items():
+            if not name.isupper():
+                raise TypeError('Settings have to be uppercase.')
```

Comment:
It would be more helpful if the name were included, e.g. "Setting 'foo' must be uppercase."

Reply:
That's a good idea, I'll change it.